common/beerocks/scripts/prplmesh_utils.sh.in

```diff
@@ -84,11 +84,17 @@ platform_deinit() {
 prplmesh_platform_db_init() {
     management_mode=${1-Multi-AP-Controller-and-Agent}
     operating_mode=${2-Gateway}
+    
+    # In certification, agent must retry onboarding indefinitely
+    certification_mode="1"
```

Comment:
Defining these variables and setting them right after is refundant, just do
```
echo "certification_mode=1"
echo "stop_on_failure_attempts=0"
```
(nitpick)

Reply:
Done